src/bp-core/compatibility/class-bb-wpml-helpers.php

```diff
@@ -224,6 +227,74 @@ public function bb_wpml_member_profile_topic_reply( $r ) {
 			return $r;
 		}
 
+		/**
+		 * Check if the post type is translatable in WPML settings.
+		 *
+		 * @since BuddyBoss [BBVERSION]
+		 * 
+		 * @param string $post_type The post type to check.
+		 * @return bool True if the post type is translatable, false otherwise.
+		 */
+		private function is_post_type_translatable( $post_type ) {
```

Comment:
Remove this function we can use below code to check post type translatable or not
			`global $sitepress;
			if ( $sitepress->is_translated_post_type( $args['post_type'] ) ) {`